data-esnext.js

```diff
@@ -271,9 +284,10 @@ exports.tests = [
     },
     {
       name: 'SIMD.%booleanType%.and',
+      spec: 'https://tc39.github.io/ecmascript_simd/#simd-and',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD/and',
       exec: function(){/*
-        return typeof SIMD.Bool32x4.and === 'function' && typeof SIMD.Bool16x8.and === 'function' && typeof SIMD.Bool8x16.and === 'function';
+        return typeof SIMD.Bool32x4.and === 'function' && typeof SIMD.Int32x4.and === 'function' && typeof SIMD.Uint32x4.and === 'function';
```

Comment:
Ah, you mean the name of the test.  i haven't noticed it was wrong.  I'll fix that in a moment.  For all operations.